src/components/PaymentCard/PaymentCard.tsx

```diff
@@ -27,10 +27,13 @@ import { PaymentCardProps } from './PaymentCard.types';
 const PaymentCard = ({ details }: PaymentCardProps) => {
   const dispatch = useAppDispatch();
   const { t } = useTranslation();
+
   const { paymentMethodSpecificParams, paymentMethod } = details;
+
   const getSpecificPaymentMethod = (): CardTypesKey => {
-    if (paymentMethod === 'card') return paymentMethodSpecificParams.variant;
-    return paymentMethod;
+    return paymentMethod === 'card' || paymentMethod === 'primer-card'
```

Comment:
I was wondering if we can somehow shorten this but the only thing that came to my mind is this:
`['card', 'primer-card'].includes(paymentMethod)` in place of `paymentMethod === 'card' || paymentMethod === 'primer-card'` but for only two card payment methods it doesnt make sense I think.